rpc/core/tx.go

```diff
@@ -212,6 +213,21 @@ func ProveShares(
 	return shareProof, nil
 }
 
+// ProveSharesV2 creates an NMT proof for a set of shares to a set of rows. It is
+// end exclusive.
```

Comment:
This Go doc seems incomplete because ProveShares also contains a binary Merkle tree proof for the relevant row roots to the data root.

Reply:
[30a7bd4](https://github.com/celestiaorg/celestia-core/pull/1351/commits/30a7bd4cf995cdea86d0cbdc304682c097d4a11d)